src/github.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+// Raw github types as per official documentation: https://docs.github.com/en/rest?apiVersion=2022-11-28
+
+namespace Github {
+	export interface Release {
```

Comment:
I remember that when using TypeScript, using interfaces does not necessarily require all fields to be listed in the interface. Therefore, my suggestion is to only list the fields you are using.